crucible-mir/src/Mir/PP.hs

```diff
@@ -135,12 +135,15 @@ pretty_temp (Var vn vm vty _vzst) =
 
 instance Pretty Fn where
     pretty (Fn fname1 fargs1 fs fbody1) =
-      vcat $ [pretty "fn" <+> pretty fname1 <> tupled (map pretty_arg fargs1)
+      vcat $ [extern <> pretty "fn" <+> pretty fname1 <> tupled (map pretty_arg fargs1)
                   <+> arrow <+> pretty rty <+> lbrace]
             ++ [indent 3 (pretty fbody1),
                 rbrace]
       where
         rty    = fs^.fsreturn_ty
+        extern = case fs^.fsabi of
+          RustAbi -> mempty
+          abi -> pretty "extern" <+> viaShow abi <+> mempty
```

Comment:
If I understand correctly, `viaShow abi` will cause `extern "rust-call" ...` to be pretty-printed as something like `extern RustCall RcNoBody ...`, which doesn't seem desirable. Perhaps we should call `pretty abi` and change the `Pretty Abi` instance to render `RustCall ...` as `"rust-call"`?

Reply:
I think the current rendering is fine.  `--print-mir` output is primarily a debugging aid and already only vaguely follows the usual Rust/MIR syntax.  Also, it's important to distinguish the `RcNoSpreadArg` and `RcSpreadArg` in the output.